pkg/rules/fmt.go

```diff
@@ -60,7 +66,10 @@ func (f *OpaFmtRule) Run(ctx context.Context, input Input) (*report.Report, erro
 						Reference:   f.Documentation(),
 					}},
 					Location: report.Location{
-						File: filename,
+						File:   filename,
+						Row:    1,
+						Column: 1,
```

Comment:
Instead of hard coding the first line, do you think we should use the location (or at least row) from `module.Package`? Could be a (metadata) comment on the first line.

Reply:
Ahh that's a good thought. I will do that tomorrow.

Reply:
I've updated this in 3697869752616f1dd7f1097f9a5e7f73bde1f0cf